### ISSUE TITLE
{cropMarks:false} disables styles passed through props

### ISSUE THREAD

**Issue by itsdanielmatos** [philcockfield/storybook-host#37]
Setting up the host with the following configs:
```
.addDecorator(
    host({
      title: "Host Container",
      width: "100%",
      height: "100%"
    })
  )
```
has the following result:
<img width="1172" alt="cropmarks enabled" src="https://user-images.githubusercontent.com/23379223/45579742-2b820280-b882-11e8-955a-34ec86eb5fcb.png">

When adding the property `cropMarks`:
```
.addDecorator(
    host({
      title: "Host Container",
      width: "100%",
      height: "100%",
      cropMarks: false
    })
  )
```
the **current result** is:
<img width="1170" alt="cropmarks disable current result" src="https://user-images.githubusercontent.com/23379223/45579810-aea35880-b882-11e8-9009-ce02fa0f8dca.png">

when it is **expected** to be:
![image](https://user-images.githubusercontent.com/23379223/45579879-3be6ad00-b883-11e8-871c-4ffccda1ea0b.png)

**Comment by itsdanielmatos** [comment]
A PR has been created #38.

**Comment by itsdanielmatos** [comment]
Closed by #38.
